spec/spec_helper_acceptance.rb

```diff
@@ -16,6 +16,7 @@
   c.add_setting :sensu_test_enterprise, default: false
   c.add_setting :sensu_manage_repo, default: true
   c.add_setting :sensu_use_agent, default: false
+  c.add_setting :skip_apply, default: false
```

Comment:
This file is pretty big to not have any docs. Please add some docs to describe what it is doing and perhaps some inline ones as you see fit.

Reply:
This particular setting isn't used, I've removed it.